webapp-django/crashstats/settings/base.py

```diff
@@ -106,6 +106,7 @@ def path(*dirs):
     "django.middleware.common.CommonMiddleware",
     "django.contrib.sessions.middleware.SessionMiddleware",
     "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "crashstats.tokens.middleware.APIAuthenticationMiddleware",
```

Comment:
As near as I can tell, we should have the authentication middlewares happen before things that look at whether the user is authenticated. If that's not true for Socorro, it's not documented anywhere. If this causes problems, I'll add a comment when I fix it.